consensus/consensus.md

```diff
@@ -150,8 +150,8 @@ nonce:
 #### Key Blocks
 
 ```
-PROTOCOL_VERSION: 25
```

Comment:
I see these protocol number update in master already, not sure why here. Though harmless so ok.